velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -122,22 +122,36 @@ inline std::string getS3BackendService(
   }
   return "Unknown";
 }
+
```

Comment:
Please add a comment explaining only Amazon (amz) and Alibaba (oss) request IDs are supported.

Reply:
done